docs/dev/community.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
```

Comment:
So ... this is a lot of duplication of the index.html. We should use https://github.com/erlydtl/erlydtl instead. LFE Dragon uses erlydtl. Check out the following:

* https://github.com/lfe/docs/tree/master/priv/templates
* https://github.com/lfex/poise (this is called in `docs-gen`)
* https://github.com/lfe/docs/blob/master/src/docs-pages.lfe (in particular, the `('landing path)` clause which generates the dev index page ... this can serve the basis of the new community page)

Let's keep this PR open, and we'll just iterate here, as you push your changes for my review.

This is a good start :-) 

(more comments below)

Reply:
Hi, i'm kind of confused because of my unfamiliarity with the tool. Can you tell me a correct way to generate new community.html? 

should I do: `(docs-pages:get-page ('landing /dev/))` ??

Reply:
It's a bit more involved; you'll have to look at how the landing page is done. I can review that, but it will be a week or two (I'm in the middle of a move and don't have the chunks of time needed to instruct on this).